Webtest.sh

```diff
@@ -61,14 +69,29 @@ docker exec $DOCKERNAME /usr/local/bin/fortio grpcping localhost
 # Do a grpcping to a scheme-prefixed destination. Fortio should append port number
 docker exec $DOCKERNAME /usr/local/bin/fortio grpcping https://fortio.istio.io
 docker exec $DOCKERNAME /usr/local/bin/fortio grpcping http://fortio.istio.io
+# Do a grpcping with -cert flag. Fortio should use valid cert.
+docker exec $DOCKERNAME /usr/local/bin/fortio grpcping -cacert $CERT fortio.istio.io:443
+docker exec $DOCKERNAME /usr/local/bin/fortio grpcping -cacert $CERT https://fortio.istio.io
 # Do a local grpcping. Fortio should append default grpc port number to destination
 docker exec $DOCKERNAME /usr/local/bin/fortio grpcping localhost
 # pprof should be there, no 404/error
 PPROF_URL="$BASE_URL/debug/pprof/heap?debug=1"
 $CURL $PPROF_URL | grep -i TotalAlloc # should find this in memory profile
+# creating dummy container to hold a volume for test certs due to remote docker bind mount limitation.
+DOCKERVOLID=$(docker create -v $TESTCERTSVOL --name certs istio/fortio.build:v6 /bin/true)
+# copying cert files into the certs volume
+for f in ca.crt server.crt server.key; do docker cp $PWD/$f certs:$TESTCERTSVOL/$f; done
+# start server in secure grpc mode. using non-default ports to avoid conflicts with fortio_server container.
+DOCKERSECID=$(docker run -d --ulimit nofile=$FILE_LIMIT --name $DOCKERSECNAME --volumes-from certs istio/fortio:webtest server -cert $TESTCERT -key $TESTKEY -grpc-port 8097 -http-port 8098 -redirect-port 8090 -loglevel $LOGLEVEL)
```

Comment:
Nope, I get error: `certificate signed by unknown authority`. It would work if we could specify a cert in the UI (which we just removed) or if cli flags were in a separate pkg, imported them into ui and assigned the cacert flag to the grpcrunner options.

Reply:
I meant add -cacert _and_ pass the value to ui.Serve but that can be a followup PR